docs/ProjectCommands.md

```diff
@@ -0,0 +1,64 @@
+# Project Command Reference
+
+Commands to do with projects. Remember that parameters with whitespace
+must be enclosed by quotation marks.
+
+## Options
+```sh
+/rocket project {list, view, help, create, unassign, edit, assign, delete}
+```
+
+### List
+```sh
+/rocket project list
+```
+
+Display a list of all projects.
+### View
+```sh
+/rocket project view PROJECT_ID
+```
+
+Displays details of project.
+
+### Help
+```sh
+/rocket project help
+```
+
+Displays options for `project` command.
+
+### Create (Team Lead and Admin only)
+```sh
+/rocket project create GH_REPO [--name DISPLAY_NAME]
+```
+
+Creates a new project from the given repo.
+### Unassign
+```sh
+/rocket project unassign PROJECT_ID
+```
+
+Unassigns the given project.
```

Comment:
unassigns from a team, or remove project from database?

Reply:
From a team. `delete` removes it from the database.

Reply:
could be a *little* clearer here 😋